blog/cncf-lfx-EAI-benchmarking/EAI-meets-KubeEdge-Ianvs.mdx

```diff
@@ -0,0 +1,494 @@
+---
+title: "Embodied Intelligence meets KubeEdge-Ianvs: Industrial Assembly Benchmarking"
+date: "2025-12-11"
+readTime: "12 min read"
+author: "Ronak Raj"
+categories : ["LFX" ]
+tags: ["KubeEdge", "Ianvs", "Embodied Intelligence","Deep Learning", "Industrial Manufacturing", "Robotic Assembly", "Force Control", "Computer Vision", "YOLOv8", "Edge AI", "Benchmarking"]
+description: "This blog introduces how to enable comprehensive embodied intelligence benchmarking for industrial manufacturing using the KubeEdge-Ianvs framework."
+---
+
+# Embodied Intelligence meets KubeEdge-Ianvs: Industrial Assembly Benchmarking
+
+This blog introduces how to enable comprehensive embodied intelligence benchmarking for industrial manufacturing using the KubeEdge-Ianvs framework.
+
+## Why Embodied Intelligence Benchmarking for Industrial Manufacturing?
+
+The manufacturing industry is experiencing a profound transformation driven by intelligent robotics, adaptive production lines, and precision assembly systems. Modern industrial environments demand more than basic task execution—they require multimodal perception, force-controlled manipulation, and real-time quality verification integrated into cohesive workflows.
+
+### A First-of-Its-Kind Benchmark Addressing Critical Gaps
+
+Despite growing interest in Embodied Intelligence for manufacturing, the research community faces a critical shortage of benchmarks that evaluate real-world industrial assembly scenarios. Existing work falls short in several key areas:
+
+#### Current Limitations in the Field:
+
+- **Isolated Task Focus**: Popular benchmarks like YCB and OCRTOC evaluate individual capabilities (grasping, object detection) but don't assess how these integrate in multi-stage industrial workflows
+- **Missing Deformable Component Coverage**: While NIST has developed benchmarks for rigid components and simple deformable objects (cables, belts), existing datasets don't support multiple representations of deformable objects with different self-occlusions typical in manufacturing [PubMed Central](https://www.ncbi.nlm.nih.gov/pmc/)
```

Comment:
https://www.ncbi.nlm.nih.gov/pmc/ redirects me to https://pmc.ncbi.nlm.nih.gov/, so why do we not have https://pmc.ncbi.nlm.nih.gov/ in here instead?